### ISSUE TITLE
Number of Columns

### ISSUE THREAD

**Issue by voteleobrown** [CodyHouse/schedule-template#1]
Not an issue, per se, but when adding two days, to make the total number 7 I'm getting a second row of days. In the situation the hours, rather than restarting for the second row, continue and split half on row one half one row two. I have not yet figured out what is dictating this behavior. 

EDIT: Nevermind, found it.

(starting at line 182 of css/style.css)

```css
@media only screen and (min-width: 800px) {
  .cd-schedule .events {
    float: left;
    width: 100%;
  }
  .cd-schedule .events .events-group {
    width: 20%; /* This 20% is the culprit. 100%/20% = 5. For 7 days, 14% works */
    float: left;
    border: 1px solid #EAEAEA;
    /* reset style */
    margin-bottom: 0;
}
```

**Comment by shores-HBI** [comment]
What was your solution to this issue? I've been having this issue too.

**Comment by voteleobrown** [comment]
See my comments in the code snippet.

**Comment by Picpool** [comment]
Is it possible to add lines adding LI ? 
When i add some , it does not work visually...

**Comment by cptncrnch** [comment]
@voteleobrown Do you know which style parameters account for the width of the horizontal lines? I also added two days to the original template (for a 7 day week) and now the bottom line of each 30-minute block has extra slack (about 20-30px) hanging past the last column. There's also 1 pixel missing from the bottom line of the row with the days of the week.

screenshot: https://imgur.com/a/KHBCPvO

update:
(line 94 of css/style.css)
````
@media only screen and (min-width: 1000px) {
  .cd-schedule .timeline li::after {
    width: calc(100% - 83px); /* timeline horizontal line width - changing this from 60px to 83px seems to work, but it still breaks when you shrink the window width so that the times listed on left-hand side disappear. */
    left: 60px;
  }
  .cd-schedule .timeline li span {
    display: inline-block;
    -webkit-transform: translateY(-50%);
        -ms-transform: translateY(-50%);
            transform: translateY(-50%);
  }
  .cd-schedule .timeline li:nth-of-type(2n) span {
    display: none;
  }
}
````

**Comment by claudia-romano** [comment]
Hi there, we have just released a new version (2.0) of this experiment (based on the [CodyHouse framework](https://codyhouse.co/ds/docs/framework)).

With this new version, you won't need to make any changes to the scss code; just make sure to add a new .cd-schedule__group element in the index.html file for each new day.

Hope this still helps.
